main.py

```diff
@@ -45,11 +45,12 @@
                    "for more verbosity.",
               type=click.IntRange(0, 3, clamp=True),
               count=True)
+@click.option("--scale-labeled-only", is_flag=True)
```

Comment:
I would make this a string-valued flag (`--scale-label=autoscale-job`). Then the presence of the label on the task (regardless of value) would trigger a rescale.